cltk/corpus/arabic/alphabet.py

```diff
@@ -0,0 +1,16 @@
+"""Arabic alphabet"""
+
+
+#Arabic digits from 0 to 9
+DIGITS = ['۰', '۱', '۲', '۳', '٤', '۵', '٦', '۷', '۸', '۹']
+
+#Diacritics
+SHORT_VOWELS = ['َ', 'ِ', 'ُ']
+
+#The Persian Alphabet
+LONG_VOWELS = ['ا', 'و', 'ي']
+
+CONSONANTS = ['ب', 'ت', 'ث', 'ج', 'ح', 'خ', 'د', 'ذ', 'ر', 'ز', 'س', 'ش', 'ص', 'ض', 'ط', 'ظ', 'ع', 'غ', 'ف', 'ق', 'ک', 'ل', 'م', 'ن', 'ه']
+
+#Special Characters
+SPECIAL = ['‎ّ', 'ﻻ']
```

Comment:
Lam-Alef has many forms , also it may be included in unicode but its not a special Arabic character for real, its a combinaition of two letters: Lam + Alef